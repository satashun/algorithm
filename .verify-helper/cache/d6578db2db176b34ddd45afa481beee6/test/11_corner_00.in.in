100000 92
0 1 5
1 1
1 2
0 2 5
1 1
1 2
1 3
0 1 5
1 1
1 2
1 3
0 2 5
1 1
1 2
1 3
1 4
1 5
0 3 1
1 1
1 2
1 3
0 3 10
1 1
1 2
1 3
0 4 12
1 1
1 2
1 3
1 4
1 5
0 3 2
1 1
1 2
1 3
1 4
1 5
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
0 10 1000000000
1 1
1 2
1 3
1 4
1 5
1 6
1 100000
0 1 1
1 1
1 2
1 3
1 4
1 5
0 1 1
1 1
1 2
1 3
1 4
1 5
1 6
0 1 1
1 1
1 2
1 3
1 4
1 5
0 1 1
1 1
1 2
1 3
1 4
1 5
1 6
0 1 1
1 1
1 2
1 3
1 4
1 5
0 2 2
1 1
1 2
1 3
1 4
1 5
