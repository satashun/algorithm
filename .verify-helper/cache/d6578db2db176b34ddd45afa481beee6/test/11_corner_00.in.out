1 5
2 0
1 5
2 5
3 0
1 10
2 5
3 0
1 10
2 10
3 0
4 0
5 0
1 10
2 10
3 1
3 11
1 10
2 10
4 12
3 11
1 10
2 10
5 0
3 13
4 12
1 10
2 10
5 0
10 10000000000
3 13
4 12
1 10
2 10
5 0
100000 0
10 10000000000
3 13
4 12
1 11
2 10
10 10000000000
3 13
1 12
4 12
2 10
5 0
10 10000000000
1 13
3 13
4 12
2 10
10 10000000000
1 14
3 13
4 12
2 10
5 0
10 10000000000
1 15
3 13
4 12
2 10
10 10000000000
1 15
3 13
2 12
4 12
